configuration/config-txt/boot.md

```diff
@@ -99,6 +99,12 @@ If `disable_splash` is set to `1`, the rainbow splash screen will not be shown o
 
 On the Raspberry Pi 4B, if this value is set to `0` then the interrupts will be routed to the ARM cores using the legacy interrupt controller, rather than via the GIC-400. The default value is `1`.
 
+## uart_2ndstage
+
+Setting `uart_2ndstage=1` causes the second-stage loader (`bootcode.bin` on devices prior to the Raspberry Pi 4, or the boot code in the  EEPROM for Raspberry Pi 4 devices) and the main firmware (`start*.elf`) to output diagnostic information to UART0. 
```

Comment:
Do we need to specify the baudrate here? Or should people just assume that it uses the same 115200 default that the Linux serial console uses?

Reply:
I think that is more detail than needed for this particular reference.